gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/synchronizer/ApiSynchronizer.java

```diff
@@ -232,7 +232,7 @@ private Maybe<ReactableApi<?>> toApiDefinition(Event apiEvent) {
             }
 
             api.setEnabled(eventPayload.getLifecycleState() == LifecycleState.STARTED);
-            api.setDeployedAt(apiEvent.getCreatedAt());
+            api.setDeployedAt(eventPayload.getDeployedAt());
```

Comment:
This is due to a change made for supporting the new v4 definition model.
Api deploy date is now in sync with the event creation date. It is preferable to add the creation date on the event created by the KubernetesSyncService ([here](https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/kubernetes/KubernetesSyncService.java#L113-L114))